apiserver: don't build a nice object with query parameters for failed requests (#943)
justtrackio/gosoline: docs/getting_started/how_to_build_an_api.md

[structmd]:# (pkg/apiserver/compression.go CompressionSettings)
**CompressionSettings**

> CompressionSettings allow the enabling of gzip support for requests and responses. By default, compressed requests are accepted, and compressed responses are returned (if suitable).

Reviewer: ```suggestion
CompressionSettings allow to enable gzip support for requests and responses. By default compressed requests are accepted and compressed responses are returned (if suitable).
```
